crates/test-fixture/src/lib.rs

```diff
@@ -801,3 +816,54 @@ impl ProcMacroExpander for Issue17479ProcMacroExpander {
         })
     }
 }
+
+// Reads ident type within string quotes, for issue #17479.
+#[derive(Debug)]
+struct Issue18898ProcMacroExpander;
```

Comment:
Would be nice if we could manage to make these defs work in line (that is where the test is defined at least) as an improvement for the tsting infra